command/plugin_reload.go

```diff
@@ -0,0 +1,104 @@
+package command
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/vault/api"
+	"github.com/mitchellh/cli"
+	"github.com/posener/complete"
+)
+
+var _ cli.Command = (*PluginReloadCommand)(nil)
+var _ cli.CommandAutocomplete = (*PluginReloadCommand)(nil)
+
+type PluginReloadCommand struct {
+	*BaseCommand
+	plugin string
+	mounts []string
+}
+
+func (c *PluginReloadCommand) Synopsis() string {
+	return "Reload mounted plugin backend"
+}
+
+func (c *PluginReloadCommand) Help() string {
+	helpText := `
+Usage: vault plugin reload [options]
+
+  Reloads mounted plugin backends. Either the plugin name or the desired plugin 
```

Comment:
This sentence is a bit confusing (specifically around "plugin backend mounts (mounts)"). Can this be paraphrased?

Reply:
Please check the new version, English is not my native language, so I'm open to suggestions :)